kratos/python/add_model_part_to_python.cpp

```diff
@@ -938,11 +962,11 @@ void AddModelPartToPython(pybind11::module& m)
         .def("IsSubModelPart", &ModelPart::IsSubModelPart)
         .def("IsDistributed", &ModelPart::IsDistributed)
         .def("AddNode", &ModelPart::AddNode)
-        .def("AddNodes",AddNodesByIds)
+        .def("AddNodes",AddNodess)
         .def("AddCondition", &ModelPart::AddCondition)
-        .def("AddConditions",AddConditionsByIds)
+        .def("AddConditions",AddConditionss)
         .def("AddElement", &ModelPart::AddElement)
-        .def("AddElements",AddElementsByIds)
+        .def("AddElements",AddElementss)
```

Comment:
if you change  it then I would remove the typos :)

Reply:
Replace is evil